0001-kbuild-AFTER_LINK.patch

```diff
@@ -17,21 +17,21 @@ Signed-off-by: Roland McGrath <roland@redhat.com>
  3 files changed, 9 insertions(+), 3 deletions(-)
 
 diff --git a/arch/arm64/kernel/vdso/Makefile b/arch/arm64/kernel/vdso/Makefile
-index b215c712d897..e18cd2a3ea53 100644
+index fa230ff09aa1..ab668b5e07cd 100644
 --- a/arch/arm64/kernel/vdso/Makefile
 +++ b/arch/arm64/kernel/vdso/Makefile
-@@ -55,7 +55,8 @@ $(obj-vdso): %.o: %.S FORCE
+@@ -48,7 +48,8 @@ $(obj-vdso): %.o: %.S FORCE
  
  # Actual build commands
- quiet_cmd_vdsold = VDSOL   $@
--      cmd_vdsold = $(CC) $(c_flags) -Wl,-n -Wl,-T $^ -o $@
-+      cmd_vdsold = $(CC) $(c_flags) -Wl,-n -Wl,-T $^ -o $@ \
-+				    $(if $(AFTER_LINK),;$(AFTER_LINK))
  quiet_cmd_vdsoas = VDSOA   $@
-       cmd_vdsoas = $(CC) $(a_flags) -c -o $@ $<
+-      cmd_vdsoas = $(CC) $(a_flags) -c -o $@ $<
++      cmd_vdsoas = $(CC) $(a_flags) -c -o $@ $< \
++			$(if $(AFTER_LINK),;$(AFTER_LINK))
```

Comment:
This looks wrong, AFTER_LINK should be after the linker call, not assembler call. How does this patch look in Fedora for kernel-5.3.x package?

Reply:
I will check the rebase. Also, Fedora does not use this patch anymore.

Reply:
In that case, I think we can drop this patch too.